src/i18n/de/ui.ts

```diff
@@ -58,16 +71,54 @@ export default UIDictionary({
 	'aside.tip': 'Tipp',
 	'aside.caution': 'Achtung',
 	'aside.danger': 'Gefahr',
+	// `<LanguageSelect>` vocabulary
+	'languageSelect.label': 'Sprache auswählen',
+  	// Integrations vocabulary
+	'integrations.changelog': 'Changelog',
+	'integrations.footerTitle': 'Weitere Integrationen',
+	'integrations.renderers': 'UI-Frameworks',
+	'integrations.adapters': 'SSR-Adapter',
+	'integrations.others': 'Sonstiges',
+  	// Checklist component
+	'checklist.or': 'oder',
+  	// Multiple Choice component
+	'multipleChoice.defaultCorrect': 'Korrekt!',
+  	'multipleChoice.defaultIncorrect': "Versuch's nochmal!",
+  	'multipleChoice.submitLabel': 'Absenden',
+  	// Tutorial Progress
+	'progress.todo': 'To-do',
+  	'progress.done': 'Erledigt',
+  	// Tutorial Navigation
+  	'tutorial.trackerLabel': 'Tutorial Tracker',
+	'tutorial.unit': 'Teil',
+  	// Tutorial
+	'tutorial.getReady': 'Mach dich bereit,…',
 	// Feedback Fish widget
 	'feedback.button': 'Gib uns Feedback',
 	'feedback.a11yLabel': 'Feedback-Formular',
 	'feedback.formTitle': 'Wie kann ich helfen?',
 	'feedback.categoryGroupLabel': 'Feedback-Typ wählen',
 	'feedback.issue': 'Problem',
+	'feedback.createIssue': 'Erstelle Github-Problem',
 	'feedback.idea': 'Idee',
 	'feedback.other': 'Sonstiges',
 	'feedback.messageA11yLabel': 'Nachricht',
 	'feedback.placeholder': 'Was sollen wir wissen?',
 	'feedback.submit': 'Feedback abschicken',
 	'feedback.close': 'Feedback-Formular schließen',
+	'feedback.success': 'Danke! Wir haben dein Feedback bekommen.',
+  	// `<FileTree>` component
+  	'fileTree.directoryLabel': 'Verzeichnis',
+  	// Code snippet vocabulary
+  	'expressiveCode.terminalWindowFallbackTitle': 'Terminal-Fenster',
+  	'expressiveCode.copyButtonTooltip': 'In Zwischenablage kopieren',
+  	'expressiveCode.copyButtonCopied': 'Kopiert!',
+  	// Backend Guides vocabulary
+  	'backend.navTitle': 'Weitere Backend-Anleitungen',
+  	// Stubs vocabulary
+	'stub.title': 'Erweitere diese Anleitung!',
+	'stub.subtitle': 'Diese Anleitung ist ein Entwurf.',
+ 	'stub.description.migration': 'Möchtest Du zu dieser Anleitung beitragen? Hast Du einen Blogbeitrag, ein Video oder eine andere Informationsquelle über die Migration von dieser Technologie zu Astro, die Du teilen möchtest?',
+	'stub.description.cms': 'Weisst Du mehr darüber, wie man dieses CMS mit Astro verwenden kann?',
+ 	'stub.description.backend': 'Weisst Du mehr darüber, wie man diesen Backend-Dienst mit Astro verwenden kann?',
```

Comment:
```suggestion
 	'stub.description.migration': 'Möchtest du zu dieser Anleitung beitragen? Hast du einen Blogbeitrag, ein Video oder eine andere Informationsquelle über die Migration von dieser Technologie zu Astro, die du teilen möchtest?',
	'stub.description.cms': 'Weißt du mehr darüber, wie man dieses CMS mit Astro verwenden kann?',
 	'stub.description.backend': 'Weißt du mehr darüber, wie man diesen Backend-Dienst mit Astro verwenden kann?',
```

Kleinere Korrekturen: Wir schreiben die persönliche Anrede "du" klein, und "weißt" wird laut Duden mit "ß" geschrieben.